repository_cli/cli/util.py

```diff
@@ -67,6 +71,23 @@ def get_records_service(data_model="rdm") -> RecordService:
     return available_services.get(data_model, current_rdm_records.records_service)
 
 
+def get_metadata_model(
+    data_model: str = "rdm", record_type: str = "record"
+) -> db.Model:
+    """Get the record model."""
+    available_models = {
+        "record": {
+            "rdm": RDMRecordMetadata,
+            "marc21": Marc21RecordMetadata,
+        },
+        "draft": {
+            "rdm": RDMDraftMetadata,
+            "marc21": Marc21DraftMetadata,
+        },
+    }
+    return available_models.get(record_type).get(data_model)
```

Comment:
should make sure to always return a db.Model
```suggestion
    return available_models.get(record_type, "record").get(data_model, "rdm")
```

Reply:
I think this is not good. It would hide a misuse of the function. It is true that a `TypeError` Exception would be raised, but for the command line and for a developer it is good to work with the `fail early and fail fast` principle.

Reply:
Then I would suggest to raise an explicit exception.